forge/forge/op/eval/forge/embedding.py

```diff
@@ -12,33 +12,33 @@ def eval(type, attr, ops):
     assert type == "embedding"
     assert len(ops) == 2
     t_ops = to_torch_operands(*ops)
-    return torch.embedding(t_ops[0], t_ops[1].to(torch.int32))
+    return torch.embedding(t_ops[1], t_ops[0].to(torch.int32))
 
 
 def shape(type, attr, ops):
     assert type == "embedding"
     assert len(ops) == 2
-    shape = list(ops[1])
-    shape.append(ops[0][-1])
+    shape = list(ops[0])
```

Comment:
Seems like this shape makes no sense now. Ops[0] is the embedding dictionary. Ops[1] is indices vector. Shape of output should be shape of ops[1] with appended ops[0][-1]. Did you ran some unit test for embeding op? I landed in this function while debugging embeding op shape mismatch in LLama attention. What happens is that you use indices like you have swapped embedding_table and indices, but you didn't. To repro, please run 
![image](https://github.com/user-attachments/assets/3450653c-a399-44be-a690-9a4e5222557a)